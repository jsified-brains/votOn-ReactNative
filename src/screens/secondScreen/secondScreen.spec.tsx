import React from 'react';
import SecondScreen from './secondScreen';
import renderer from 'react-test-renderer';

describe('SecondScreen component.', () => {
  it('matches expected snapshot', () => {
    const header = renderer.create(
      <SecondScreen />
    ).toJSON;
    expect(header).toMatchSnapshot();
  });

  it('renders without crashing', () => {
    const rendered = renderer.create(<SecondScreen />).toJSON();
    expect(rendered).toBeTruthy();
  });
});
