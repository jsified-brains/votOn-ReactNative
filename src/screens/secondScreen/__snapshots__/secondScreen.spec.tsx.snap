// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SecondScreen matches expected snapshot 1`] = `[Function]`;
